nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py

```diff
@@ -1,8 +1,29 @@
 """Authors: Cody Baker and Ben Dichter."""
-from spikeextractors import SpikeGLXRecordingExtractor
+from spikeextractors import SpikeGLXRecordingExtractor, SubRecordingExtractor
 
 from ..baserecordingextractorinterface import BaseRecordingExtractorInterface
 
 
 class SpikeGLXRecordingInterface(BaseRecordingExtractorInterface):
     RX = SpikeGLXRecordingExtractor
+
+    @classmethod
+    def get_input_schema(cls):
+        input_schema = super().get_input_schema()
+        input_schema['properties'].update(sync_with_ttl=dict(type='string', default=True))
```

Comment:
Seems like the type should be boolean ([ref](https://json-schema.org/understanding-json-schema/reference/boolean.html))

Reply:
Correct, fixed. Although, check comment below.